deploy/deploy.sh

```diff
@@ -2,7 +2,9 @@
 
```

Comment:
Changes to deploy.sh must be manually deployed right now.

Reply:
Do you know of a strategy for putting the `deploy/*` files in `.circleci/` and/or otherwise including changes to deploy scripts in the CircleCI build process?